docs/governance/decisions/929-move-repository.md

```diff
@@ -27,51 +27,91 @@ Moving the `earthaccess` repo to another GitHub organization will:
+5. PyPI and conda-forge releases:
+   * earthaccess publication to both PyPI and conda-forge package managers should continue as expected without any breaking changes.
+
+This transfer would lead to the following administrative changes:
+  
+1. Local Clones:
+   * According to GitHub [Transferring a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository) documentation, "All links to the previous repository location are automatically redirected to the new location. When you use git clone, git fetch, or git push on a transferred repository, these commands will redirect to the new repository location or URL. However, to avoid confusion, we strongly recommend updating any existing local clones to point to the new repository URL. You can do this by using git remote on the command line: `git remote set-url origin NEW_URL`
+   * We may want to consider if/how to provide guidance to our community on this recommended update, though we expect this update would cause minimal disruption.
+3. Permissions and Access to the new organization:
+   * earthaccess maintainers would need to consider updating/reconfiguring permissions and access settings for team members within the new organization.
+   * We may need to consider updates to our existing contributor documentation to reflect any applicable changes to our access processes.
+3. Integrations and Third-Party Tools:
+   * This may not apply to earthaccess, but we ought to consider whether any existing integrations in the NSIDC GitHub organization apply and would need to be re-connected to the migrated repository. 
+4. GitHub Project configuration
+   * Classic GitHub Projects tied to the repository will not transfer and references to issues/PRs within them may break. We need to identify whether we are utilizing classic vs new (beta) projects. For the latter option, project configuration may remain but may need manual re-association. 
 
 
 ## Considered Options
 
-?
+### Option 1: Don't move; stay within `nsidc` org.
 
+Pros:
+* No migration effort (and associated disruption) needed
 
-## Decision Outcome
+Cons:
+* No clear path to extend organizational ownership permissions to earthaccess maintainers outside of NSIDC
 
-?
+### Option 2: Move to an independent org, e.g. `earthaccess-dev`.
 
+Pros:
+* Overall benefits described above
+* Full / flexible community governance
+* Custom branding?
 
-### Option 1
+Cons:
+* Migration effort
+* Potential for reduced visibility without institutional org backing?
 
-Don't move; stay within `nsidc` org.
 
-### Option 2
+### Option 3: Move to a sponsor / incubator org, e.g. `pangeo`, `openscapes`.
+
+Pros:
+* Overall benefits described above
+* Built-in open source credibility and visibility
+* Leverage existing communities, increased contributor base?
+* Leverage existing software infrastructure?,
+* Leverage existing governance models?
+* Potential funding opportunities?
```

Comment:
I think we get all of the above from e.g. going through pyopensci review without being in a special org. I wouldn't describe these benefits as unique to being in a specific org.